src/hooks.js

```diff
@@ -51,3 +50,27 @@ export const useMultipleChoice = (initialValue) => {
   };
   return { value, onChange: handleChange };
 };
+
+export const useFormInput = (initialValues) => {
+  const [values, setValues] = useState(initialValues);
+  const handleChange = (e) => {
+    setValues({
+      ...values,
+      [e.target.name]: e.target.value,
+    });
+  };
+
+  return [values, handleChange];
+};
+
+export const useBoolean = (initialValue) => {
+  const [value, setState] = useState(initialValue);
+  const setTrue = () => {
+    setState(true);
+  };
+  const setFalse = () => {
```

Comment:
awkward si tbh, usestate aja utk bagian logged in or not
useBoolean itu dipake utk checkbox yg true or false lgsg dicontrol user